### ISSUE TITLE
Armor pen does not affect Aimed Shot properly

### ISSUE THREAD

**Issue by chickennugger** [dalaranwow/dalaran-wow#5446]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Currently, ArP does not affect Aimed shot. Additionally, there is an error in the calculation of armor's damage reduction for aimed shot Crits. 

currently the damage done by my aimed shot HITS (with ~27% ArP) are what i should expect to observe with 0% ArP (~58% damage  from pre armor calculated damage). My Aimed Shot crits are doing damage consistent with a damage reduction % similar to 0% ArP + an additional 10% damage reduction (~48% damage from pre armor calculated damage)

See the images I provided in the proof section.  for the data to back this up.

**Expected behaviour**: Tell us what should happen instead

before I do the aimed shot math, lets start with armor/ArP:
DR% = Armor / (Armor + 15,232.5)
Effective Armor(post ArP)= Armor-(ArP x Cap)=
Cap= (10643+(400+85 x 83+4.5 x 85*(83-59)))/3 = 9092.66

merging it all together:
DR% = (10643-ArP x 9092.66)/((10643-ArP x 9092.66)+15232.5)

Now, 
if you have 0% ArP, bosses armor (10643) would reduce your damage by
10643/(10643+15232.5) = **41.11316%**
this means that when you run the calculation to see what damage you do, you must multiply your pre armor damage by (1-41.11316%) = **58.0064%**

in the gear I was testing with( no trinkets/set bonuses that could modify the size of hits/crits), I had 

for the sake of succinctness, i'm only going to show the calculations for the average hit/crit. when i was testing however, i calculated the damage range as well (min, max).

aimed shot damage SHOULD be:

HITS:
(12% from barrage, 5% from ranged weapon spec)
(avg char sheet damage + 456.5) x 1.12 x 1.05 x (1-DR%)
expanding out the armor portion of the equation:

CRITS:
(100% from base crit dmg bonus, 30% from mortal shots, 10% from marked for death, plus the above multipliers)
(avg char sheet damage + 456.5) x 1.12 x 1.05 x 2 x 1.3 x 1.1 x (1-DR%)

**Steps to reproduce the problem**:

1. calculate expected damage for aimed shot
2. go to boss target dummy, use aimed shot a bunch to get a sample
3. compare

**Include proofs for this behaviour**

pages used to get the formulas,etc for armor/ArP
http://wowwiki.wikia.com/wiki/Damage_reduction
http://wowwiki.wikia.com/wiki/Armor_penetration

![wowscrnshot_050418_184150](https://user-images.githubusercontent.com/38992413/39657150-dee5d408-4fca-11e8-937b-3f2c66c42503.jpg)

![hunter arp math](https://user-images.githubusercontent.com/38992413/39657140-bfaca1e8-4fca-11e8-9e79-7d98e16b5ae2.PNG)

I can provide the spreadsheet shown above too if that would help.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Kennehsauce** [comment]
Awesome break down, baller af.

**Comment by hrtyv** [comment]
Really nice effort on this thread @chickennugger, nice amount of details and calculations. 
This should help the staff getting something done about this issue, there have been some issues posted but lacked proof and details about the issue itself. 
I hope this will be the deal breaker for hunters to be able to benefit from ArP to their full potential again.

@TeridanDW @Lothloryen 
This issue should cover the missing proof of the earlier mentions of this problem. Issue#4902 Issue#5397

Good job.

**Comment by crulinjr** [comment]
Hey guys on my server ppl tell me that Mortal Shots and Marked for Death adds additively and not multiplicatively so it owuld be  average Aimed shot * 2.4 instead of the *2*1.3*1.1

**Comment by Kennehsauce** [comment]
Rip, no response from devs.

**Comment by Kookaburra129** [comment]
Bump. Request some attention here so hunters can break out of being threat bots while having end-game gear.

**Comment by giammafix** [comment]
Bump. It seems that the issue hasn't been solved yet.

**Comment by Kookaburra129** [comment]
@TeridanDW @Lothloryen Please can we get some sort of response to know if this issue is even being looked at? Hope we all agree that it can be considered game breaking (for hunter mains).

-Finalshot

**Comment by Kookaburra129** [comment]
Bump, requesting attention.

**Comment by Kennehsauce** [comment]
@TeridanDW @Lothloryen plz halp

**Comment by Undertower** [comment]
I feel like this report a bit off as the formula used for your Aimed Shot is wrong.
You wrote:
(avg char sheet damage + 456.5) x 1.12 x 1.05 x (1-DR%)

When the correct formula for average damage is: ((min damage+max damage/2) + (2.8 * Attack Power / 14)) + (ammo damage * 2.8) + 408= x 1.12 x 1.05 x (1-DR%)
